requirements.txt

```diff
@@ -1,6 +1,6 @@
 six
 Jinja2
-slackclient>=1.3.0,<2.0.0
+slackclient>=2.0.0
```

Comment:
**suggestion (non-blocking)** I believe that we don't need to specify a minimum version here, as the latest version of this library should work correctly now.